src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -76,38 +76,42 @@ const QUERY = gql`
 
 type Transcript = {
   stable_id: string;
-  cross_references: CrossReference[];
+  external_references: ExternalReferenceType[];
 };
 
 type Gene = {
   name: string;
   stable_id: string;
   transcripts: Transcript[];
-  cross_references: CrossReference[];
+  external_references: ExternalReferenceType[];
 };
 
-const buildCrossReferenceGroups = (crossReferences: CrossReference[]) => {
-  const crossReferenceGroups: { [key: string]: CrossReferenceGroup } = {};
+const buildExternalReferencesGroups = (
+  externalReferences: ExternalReferenceType[]
+) => {
+  const externalReferencesGroups: {
+    [key: string]: ExternalReferencesGroup;
+  } = {};
 
-  crossReferences.forEach((crossReference) => {
-    const sourceId = crossReference.source.id;
+  externalReferences.forEach((externalReference) => {
+    const sourceId = externalReference.source.id;
 
-    if (!crossReferenceGroups[sourceId]) {
-      crossReferenceGroups[sourceId] = {
-        source: crossReference.source,
+    if (!externalReferencesGroups[sourceId]) {
```

Comment:
Should it be `externalReferenceGroups`? I don't mind the plural form of `externalReference` either, but just checking if this was unintentional.

Reply:
It was intentional 😃  I debated with myself whether the singular or the plural form sounded better for a group of external references, and convinced myself that the plural form looked better.

Now that I'm looking for examples of usage, I am seeing that in the .Net community they have xml tags named ItemGroup and PropertyGroup, both singular:

![image](https://user-images.githubusercontent.com/6834224/96469574-45daf700-1225-11eb-979d-8b9a7b0f7302.png)

On the other hand, we have TranscriptsList and ProteinsList.

Reply:
That's a good example of an external project naming these variables. Since we are already using the plural form for the lists, should we then agree to do the same for groups too? Think using one form is better and will avoid confusion.